src/Media/ImageUploaderService.php

```diff
@@ -47,7 +48,7 @@ public function __construct(
         int $maxFileSize = null
     ) {
         if ($maxFileSize < 0) {
-            throw new \RuntimeException('Max file size should be 0 or bigger inside config.yml.');
+            throw new RuntimeException('Max file size should be 0 or bigger inside config.yml.');
```

Comment:
```suggestion
            throw new RuntimeException('Max file size should be 0 or bigger inside config.php.');
```

Reply:
Best to use `config.php` for clarity

Reply:
I've updated it so it's agnostic of the config file name, since anything in `src` should not be concerned with how the app is bootstrapped.